client/battle/BattleInterfaceClasses.cpp

```diff
@@ -446,6 +446,119 @@ void HeroInfoBasicPanel::show(Canvas & to)
 	CIntObject::show(to);
 }
 
+
+StackInfoBasicPanel::StackInfoBasicPanel(const CStack * stack, Point * position, bool initializeBackground)
+	: CIntObject(0)
+{
+	OBJECT_CONSTRUCTION_CAPTURING(255-DISPOSE);
+	if (position != nullptr)
+		moveTo(*position);
+
+	if(initializeBackground)
+	{
```

Comment:
Why is this option needed?

Reply:
same as above

Reply:
if I got to blind guess then some differences depending on if resolution is 800x600 or bit higher and stuff render bit differently in that case (there was something about rendering outside battle window)